backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/OutputGenerator.java

```diff
@@ -93,8 +95,10 @@ public String getComponentLongName(LicenseInfoParsingResult li) {
 
     public VelocityContext getConfiguredVelocityContext() {
         Properties p = new Properties();
-        p.setProperty("resource.loader", "class");
-        p.setProperty("class.resource.loader.class", "org.apache.velocity.runtime.resource.loader.ClasspathResourceLoader");
+        p.setProperty(RuntimeConstants.RESOURCE_LOADER, "file, class");
+        p.setProperty(RuntimeConstants.FILE_RESOURCE_LOADER_PATH, "/etc/sw360");
```

Comment:
`"/etc/sw360"` is surely already defined as a constant. Like at `CommonUtils.SYSTEM_CONFIGURATION_PATH`, for example. You only have to make it public.

Reply:
Oh, and while you're at it, make it `final`, please.

Reply:
changed.
I set the variable SYSTEM_CONFIGURATION_PATH to public and final.